.stylelintrc

```diff
@@ -0,0 +1,68 @@
+{
+  "extends": "stylelint-config-recommended",
+  "plugins": [
+    "stylelint-declaration-block-no-ignored-properties",
+    "stylelint-order",
+    "stylelint-scss"
+  ],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "block-no-empty": true,
+    "color-hex-case": "lower",
+    "color-hex-length": "long",
+    "color-named": "never",
+    "comment-no-empty": true,
+    "declaration-block-no-shorthand-property-overrides": true,
+    "declaration-block-no-duplicate-properties": true,
+    "declaration-block-no-redundant-longhand-properties": true,
+    "declaration-block-trailing-semicolon": "always",
+    "declaration-no-important": true,
+    "font-family-name-quotes": "always-unless-keyword",
+    "font-family-no-duplicate-names": true,
+    "font-weight-notation": "numeric",
+    "function-calc-no-invalid": true,
+    "function-calc-no-unspaced-operator": true,
+    "function-comma-space-after": "always",
+    "function-name-case": [
+      "lower",
+      ignoreFunctions: ["containerWidth", "lineHeight", "zIndex", "fontSizesByBreakpoint", "withNElements"]
+    ],
+    "function-url-quotes": "always",
+    "function-whitespace-after": "always",
+    "indentation": 2,
+    "length-zero-no-unit": true,
+    "max-nesting-depth": 2,
+    "no-descending-specificity": true,
+    "no-duplicate-at-import-rules": true,
+    "no-duplicate-selectors": true,
+    "no-extra-semicolons": true,
+    "no-invalid-double-slash-comments": true,
+    "no-unknown-animations": true,
+    "order/properties-alphabetical-order": true,
```

Comment:
ho sempre preferito un ordine che ricordasse quello del box-model, ad esempio [questo](https://github.com/constverum/stylelint-config-rational-order), che ne dite?

Reply:
Lo preferisco anche io, ma devo dire con le proprietà in ordine alfabetico ci sono meno errore, alla fine è questione di abitudine.

Piuttosto, i vari plugin che fanno queste cose ordinano male, mettendo tipo:
`padding-left: 10px;`
prima di 
`padding: 20px;`
e quindi si rompe, questo come lo fa?

Reply:
Il loro readme dice "Shorthand properties must always precede their longhand counterparts, even if that means they are not alphabetized. (See also declaration-block-no-shorthand-property-overrides.)"
Mi fido che ci abbiano pensato insomma 🤔

Reply:
Si comporta esattamente uguale basta guardare questo file https://github.com/constverum/stylelint-config-rational-order/blob/master/groups/boxModel.js

Per esperienza l'ordinamento alfabetico risulta quello più chiaro e porta a duplicare meno proprietà mantenendo consistenza mel blocco di style